components/email/src/Email.svelte

```diff
@@ -500,6 +504,125 @@
     });
   }
 
+  async function handleReplyAllClick(event: MouseEvent, msgIndex: number) {
+    event.stopImmediatePropagation();
+
+    const currentMessage = activeThread.messages[msgIndex];
+
+    const me: Participant = {
+      name: _this.you.name,
+      email: _this.you.email_address,
+    };
+    const participantsWithoutMe = activeThread.participants.filter(
+      (e) => e.email != me.email,
+    );
+
+    const from = [me];
+    let to = participantsWithoutMe;
+    const subject = currentMessage.subject?.toLowerCase().startsWith("re:")
+      ? currentMessage.subject
+      : `Re: ${currentMessage.subject}`;
+
+    const value = {
+      reply_to_message_id: currentMessage.id,
+      from: from,
+      to: to,
+      reply_to: from,
+      subject: subject,
+    };
+
+    if (_this.with_composer) {
+      dispatchEvent("replyAllClicked", {
+        event,
+        message: activeThread.messages[msgIndex],
+        thread: activeThread,
+        value,
```

Comment:
Which props should be in `value` vs in the top level object? It looks like some of them appear in both (to, from, subject). Can we just have everything in `value` and just emit an event with `{ event, value }`?